Docs/source/Software/Apps.rst

```diff
@@ -173,6 +172,18 @@ Usage
 Additional Considerations
     None.
 
+Voltage
+========
+
+Purpose
+    This App is used to keep track of the voltages of all the modules in the battery pack.
+
+Usage
+    ``Voltage_Init()`` must be called before calling any other Voltage functions. Descriptions of
+    other functions are located in Voltage.h.
+
+Additional Considerations
+    None
```

Comment:
I would mention in one of these sections that all of the functions that do calculations based on voltage measurements operate on a buffer that is only updated by calling ``Voltage_UpdateMeasurements()`` since this is not obvious from reading the header file